Week2/arrays.js

```diff
@@ -0,0 +1,8 @@
+var myArrys = [];
+console.log("arrays are used to store multiple values in a single variable.");
```

Comment:
yeah that is correct, but what do you think the value of the `myArrs`?

Reply:
now i made it .